scripts/create_tables.sql

```diff
@@ -1 +1,33 @@
 -- create doctors, patients, and appointments tables
+create table doctors (
+  id serial primary key,
+  surname varchar,
+  given_name varchar,
+  gender varchar,
+  dob date,
```

Comment:
I don't think that DATE will accept that format found in the dob column of the .csv file. VARCHAR or TEXT would work.